hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheBatchInvalidationTest.java

```diff
@@ -153,7 +153,7 @@ public void testMapClear_shouldClearNearCaches_onOwnerAndBackupNodes() throws Ex
         String mapName = randomMapName();
         Config config = newConfig(mapName);
         config.setProperty(GroupProperty.PARTITION_COUNT.getName(), "1");
-        configureBatching(config, true, Integer.MAX_VALUE, Integer.MAX_VALUE);
+        configureBatching(config, true, 5, 5);
```

Comment:
Since these two parameters are the `hazelcast.map.invalidation.batch.size` and `hazelcast.map.invalidation.batchfrequency.seconds` and `Integer.MAX_VALUE` basically should configure this system unusable, I guess it was not used before (due to the distinct operations), and know it is used and needs a more realistic configuration to work.